grails-app/services/au/org/emii/gogoduck/job/JobExecutorService.groovy

```diff
@@ -1,11 +1,82 @@
 package au.org.emii.gogoduck.job
 
+import au.org.emii.gogoduck.worker.Worker
+
+import java.util.concurrent.LinkedBlockingQueue
+import java.util.concurrent.BlockingQueue
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
 class JobExecutorService {
 
+    static BlockingQueue<Map> JOB_QUEUE
```

Comment:
Not so much an issue of effort per se, but it does raise a few questions, namely:

* what should the limit be?
* what happens when the queue is full and a user requests a job?

Given that this task was only scored a 3 in total, perhaps it would be a good idea to get this on the backlog as a separate item?

Tagging @pblain for his opinion.